lib/index.js

```diff
@@ -417,24 +419,29 @@ Utils.cloneTerse = function(input) {
  *  - -4d -6h will reduce the parsed date by 4 days and 6 hours
  *
  * @param  {String} field the date string to parse.
+ * @param  {String} locale the locale string OR options object.
  * @return {Object}       an object containing the parsed date, or the passed field if it was not a String.
  */
-Utils.parseDateTimeField = function(field) {
+Utils.parseDateTimeField = function(field, locale) {
   var getRtnObject = function(parsed, isValid) {
     return formatParsedObject('date', field, isValid, parsed);
   };
 
-  var parsedDate;
+  var options = {};
+  if (_.isString(locale)) {
+    options.locale = locale;
+  } else if (locale) {
+    options = locale;
+  }
```

Comment:
My original thinking was to mimic the second param of `SugarDate.create` which behaves like this. But I do like the idea of not having a mixed type as a parameter.